codecommit-put-file-action/README.md

```diff
@@ -0,0 +1,61 @@
+# Action to put a file in a Codecommit repo 🔄
+
+This action uses the [vanilla AWS CLI](https://docs.aws.amazon.com/cli/index.html) to put file in the codecommit repo.
+
+
+## Usage
+
+### `workflow.yml` Example
+
+Place in a `.yml` file such as this one in your `.github/workflows` folder. [Refer to the documentation on workflow YAML syntax here.](https://help.github.com/en/articles/workflow-syntax-for-github-actions)
+
+```
+name: Codecommit put-file
+on: push
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@master
+    - name: Put file into a codecommit repo
+      uses: aws-robotics/robomaker-sample-app-ci/codecommit-put-file-action@v2.0.0
+      env:
+        AWS_REGION: 'us-east-2'
+        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        AWS_CODECOMMIT_REPO_NAME: ${{ secrets.REPO_NAME }}
+        AWS_CODECOMMIT_BRANCH_NAME: ${{ secrets.BRANCH_NAME }}
+        DEST_FILE_CONTENT: {"name": "hello-world"}
```

Comment:
I think the example should just be a normal string here (should it be quoted or not?). I think using JSON syntax here is confusing; the user may think the content must be in JSON format.

Reply:
Good point..I think the description below is a better reference for the user.. however, to reduce the ambiguity you pointed out, I have updated the README with the sample file ref and env var descriptions from [codecommit documentation](https://docs.aws.amazon.com/cli/latest/reference/codecommit/put-file.html).